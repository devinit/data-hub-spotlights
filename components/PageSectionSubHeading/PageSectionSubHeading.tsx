import React, { FunctionComponent } from 'react';

const PageSectionSubheading: FunctionComponent = ({ children }) => {
  return <h3 className="section__sub-heading">{children}</h3>;
};

export { PageSectionSubheading };
