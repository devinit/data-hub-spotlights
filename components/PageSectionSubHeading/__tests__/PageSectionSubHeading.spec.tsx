/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { PageSectionSubheading } from '../PageSectionSubHeading';

describe('PageSectionSubheading', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<PageSectionSubheading>Child</PageSectionSubheading>).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
