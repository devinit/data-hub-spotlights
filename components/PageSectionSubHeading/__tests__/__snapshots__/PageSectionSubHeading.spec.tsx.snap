// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSectionSubheading renders correctly 1`] = `
<h3
  className="section__sub-heading"
>
  Child
</h3>
`;
